contrib/packs/actions/install.meta.yaml

```diff
@@ -21,5 +21,5 @@
       default: "master"
     register:
       type: "string"
-      default: "actions"
+      default: "actions,aliases,sensors"
```

Comment:
Oh, they do? Cool, I didn't know that.

(I do see the `CAST_OVERRIDES` code now...)